airbyte-config/init/src/main/resources/seed/destination_definitions.yaml

```diff
@@ -93,7 +93,7 @@
     normalizationRepository: airbyte/normalization-clickhouse
     normalizationTag: 0.4.0
     normalizationIntegrationType: clickhouse
-  supportsDbt: true
+  supportsDbt: false
```

Comment:
did clickhouse not support `dbt` or are we turning off normalization for clickhouse here?

Reply:
Clickhouse does not actually support custom `dbt` transformations and clickhouse `spec.json` has `supportsDbt` set to `false`. So there was a discrepancy between `destination_definitions.yaml `and `spec.json`. My guess is this happened with normalization configs were moved to `destination_definitions.yaml` via a script (@evantahler is this a fair assumption?)